tests/CommunityToolkit.Mvvm.ExternalAssembly/CommunityToolkit.Mvvm.ExternalAssembly.csproj

```
@@ -4,8 +4,11 @@
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
-  <!-- Manually apply the properties of test projects to this assembly as well,
-       which is neither a test project not one of the libraries to pack for NuGet. -->
+  <!--
+      MVVM unit test uses this project to test it as a `ProjectReference`.
+      Since, we don't need to pack or publish the project, disable those
+      targets and ignore validation warnings given in those contexts.
```

Drop the ` as they don't do anything in XML.
Also, please keep comments impersonal.
```suggestion
      The MVVM Toolkit unit tests use this project to test it as a ProjectReference.
      Since this project is neither packed nor published, those targets can be
      disabled, and validation warnings can be ignored in those contexts.
```

---

The backtick I use in comments to represent code references, even though, the editors can't do anything now but it's future proof. Also, one of the extensions I use actually highlight this separately.

---

Please just don't use ` and remove them from the places where you've added that 🙂
It's not about future proofing, it's just not part of XML, so it shouldn't be there.

---

XML text doesn't have a specific restriction on symbols but if you don't like backtick, I'll replace it with standard single quote as I did in other places. This was just an experiment to get feedback.

---

I would appreciate if you could just follow the review feedbacks here, as it'd save a lot of back and forth. I've asked you multiple times to just leave those identifiers as they were. Can you please just do that? Don't add anything.

---

Sure. I was only trying to improve the readability.